soroban-sdk/src/token.rs

```diff
@@ -0,0 +1,97 @@
+//! Token contains types for calling and accessing token contracts, including
+//! the Stellar Asset Contract.
+//!
+//! See [`Interface`] for the interface of token contracts such as the Stellar
+//! Asset Contract.
+//!
+//! See [`Client`] for calling token contracts such as the Stellar Asset
```

Comment:
Should this say "Use Client" instead of "See Client"? I was looking around for a Client because of this client.

Reply:
```suggestion
//! Use [`Client`] for calling token contracts such as the Stellar Asset
```